experimental/presentation/presentation_api.js

```diff
@@ -70,75 +104,112 @@ function removeEventListener(name, callback) {
   }
 }
 
-function handleDisplayAvailableChange(isAvailable) {
+// APIs
+exports.startSession = function(presentationUrl) {
+  return new Promise(function(resolve, reject) {
+    var requestId = ++_nextRequestId;
+    var request = new SessionRequest(requestId, resolve, reject);
+    _sessionRequests[requestId] = request;
+
+    var baseUrl = location.href.substring(0, location.href.lastIndexOf("/")+1);
```

Comment:
And refer to http://stackoverflow.com/questions/1420881/how-to-extract-base-url-from-a-string-in-javascript, I prefer to use `location.protocol + "//" + location.host`